src/token.js

```diff
@@ -70,32 +70,104 @@ class OpenPAYGOTokenShared {
     }
 
     static convertFrom4digitToken(token) {
-        // 4 - digit token value
-        let bitArray = []
-        for (let digit of token.toString()) {
-            digit = Number(digit) - 1
-            const tempArr = bitArrayFromInt(digit, 2)
-            bitArray = bitArray.concat(tempArr)
-        }
-        return bitArrayToInt(bitArray)
+        return convertFrom4digitToken(token)
     }
```

Comment:
Is it required to have the function `convertFrom4digitToken` twice? As a class function and as a global? This functional declaration seems a bit odd to the viewer.

Reply:
It does look odd, but the reason for it is to avoid duplicating the function in both classes (shared/extended)

Reply:
Ok, understood 🎉